bokeh/server/views/session_handler.py

```diff
@@ -79,17 +79,23 @@ def initialize(self, *args, **kw):
                 raise HTTPError(status_code=403, reason="No bokeh-session-id provided")
 
         if token is None:
-            if self.application.request_headers is None:
-                headers = list(self.request.headers)
+            if self.application.include_headers is None:
+                allowed_headers = list(self.request.headers)
             else:
-                headers = self.application.request_headers
-            headers = {k: v for k, v in self.request.headers.items() if k in headers}
-
-            if self.application.request_cookies is None:
-                cookies = list(self.request.cookies)
+                allowed_headers = self.application.include_headers
+            allowed_headers = [header for header in headers if header not in
+                               self.application.exclude_headers]
+            headers = {k: v for k, v in self.request.headers.items()
+                       if k in allowed_headers}
```

Comment:
This could be OK, tho I wonder want the actual use case is for supplying both. Other options:

* one of include/exclude takes absolute precedence over the other, in case both are provided 
* providing both raises an error

Reply:
I don't think there is any such use case but I didn't see any particular harm in allowing both. Happy to raise an error though due to the potential ambiguity.

Reply:
My preference would be to raise an error in case both are provided.